tests/manifold/spherical_manifold_07.output

```diff
@@ -0,0 +1,123 @@
+
+set view equal xyz
+set size ratio -1
+set multiplot
+plot '-' with vectors 
+-0.707107 -0.707107 -0.0707107 -0.0707107
+-1.83697e-16 -1.00000 -1.83697e-17 -0.100000
+-1.83697e-16 -1.00000 -1.83697e-17 -0.100000
+0.707107 -0.707107 0.0707107 -0.0707107
+-0.707107 -0.707107 0.0707107 0.0707107
+-1.00000 1.22465e-16 0.100000 -1.22465e-17
+-1.00000 1.22465e-16 0.100000 -1.22465e-17
+-0.707107 0.707107 0.0707107 -0.0707107
+0.707107 -0.707107 0.0707107 -0.0707107
+1.00000 0.00000 0.100000 0.00000
+1.00000 0.00000 0.100000 0.00000
+0.707107 0.707107 0.0707107 0.0707107
+-0.707107 0.707107 0.0707107 -0.0707107
+6.12323e-17 1.00000 -6.12323e-18 -0.100000
+6.12323e-17 1.00000 -6.12323e-18 -0.100000
+0.707107 0.707107 -0.0707107 -0.0707107
+e
+set view equal xyz
+set size ratio -1
+set multiplot
+splot '-' with vectors 
+-0.577350 -0.577350 -0.577350 0.0577350 0.0577350 0.0577350
+0.00000 -0.707107 -0.707107 1.68252e-17 0.0707107 0.0707107
+-0.707107 0.00000 -0.707107 0.0707107 0.00000 0.0707107
+0.00000 0.00000 -1.00000 1.22465e-17 0.00000 -0.100000
+0.00000 -0.707107 -0.707107 1.11038e-17 0.0707107 0.0707107
+0.577350 -0.577350 -0.577350 -0.0577350 0.0577350 0.0577350
+0.00000 0.00000 -1.00000 -1.22465e-17 0.00000 0.100000
+0.707107 0.00000 -0.707107 -0.0707107 3.18167e-18 0.0707107
+-0.707107 0.00000 -0.707107 0.0707107 -8.52818e-18 0.0707107
+0.00000 0.00000 -1.00000 -1.22465e-17 0.00000 0.100000
+-0.577350 0.577350 -0.577350 0.0577350 -0.0577350 0.0577350
+0.00000 0.707107 -0.707107 5.60839e-18 -0.0707107 0.0707107
+0.00000 0.00000 -1.00000 -1.22465e-17 0.00000 0.100000
+0.707107 0.00000 -0.707107 -0.0707107 -3.18167e-18 0.0707107
+0.00000 0.707107 -0.707107 -6.66229e-18 -0.0707107 0.0707107
+0.577350 0.577350 -0.577350 -0.0577350 -0.0577350 0.0577350
+0.577350 -0.577350 -0.577350 -0.0577350 0.0577350 0.0577350
+0.707107 -0.707107 0.00000 -0.0707107 0.0707107 -6.12323e-18
+0.707107 0.00000 -0.707107 -0.0707107 3.18167e-18 0.0707107
+1.00000 0.00000 0.00000 -0.100000 0.00000 -6.12323e-18
+0.707107 0.00000 -0.707107 -0.0707107 -3.18167e-18 0.0707107
+1.00000 0.00000 0.00000 -0.100000 0.00000 -6.12323e-18
+0.577350 0.577350 -0.577350 -0.0577350 -0.0577350 0.0577350
+0.707107 0.707107 0.00000 -0.0707107 -0.0707107 -6.12323e-18
+0.707107 -0.707107 0.00000 -0.0707107 0.0707107 -6.12323e-18
+0.577350 -0.577350 0.577350 -0.0577350 0.0577350 -0.0577350
+1.00000 0.00000 0.00000 -0.100000 0.00000 -6.12323e-18
+0.707107 0.00000 0.707107 -0.0707107 -3.18167e-18 -0.0707107
+1.00000 0.00000 0.00000 -0.100000 0.00000 -6.12323e-18
+0.707107 0.00000 0.707107 -0.0707107 3.18167e-18 -0.0707107
+0.707107 0.707107 0.00000 -0.0707107 -0.0707107 -6.12323e-18
+0.577350 0.577350 0.577350 -0.0577350 -0.0577350 -0.0577350
+-0.577350 -0.577350 0.577350 0.0577350 0.0577350 -0.0577350
+-0.707107 0.00000 0.707107 0.0707107 -4.72716e-17 -0.0707107
+0.00000 -0.707107 0.707107 5.60839e-18 0.0707107 -0.0707107
+0.00000 0.00000 1.00000 nan nan nan
```

Comment:
This is not related to the current PR, but with `SphericalManifold`. All of these tests are really stressing our most used manifolds...